spec/hoosegow_docker_spec.rb

```diff
@@ -33,17 +32,39 @@
         "/inside/path" => "/home/burke/data-for-container:rw",
         "/other/path" => "/etc/shared-config",
       } }
-      its(:volumes_for_create) { should == {
-        "/inside/path" => {},
-        "/other/path" => {},
-      } }
       its(:volumes_for_bind) { should == [
         "/home/burke/data-for-container:/inside/path:rw",
         "/etc/shared-config:/other/path:ro",
       ] }
     end
   end
 
+  context "volume is writable" do
+    it "calls after_create" do
+      test_dir = File.join(Dir.pwd, 'volume-test')
+      FileUtils.remove_dir(test_dir, true)
+      FileUtils.mkpath(test_dir)
+
+      config = CONFIG.merge(
+        :Entrypoint => ['/usr/bin/touch',  '/volume-test/test'],
+        :volumes => { '/volume-test' => test_dir + ":rw"}
```

Comment:
Barely remembering what is going on here... funny that this is `:volumes` instead of `:Volumes` – is that intentional?

Reply:
Yep. The api is such that lower case keys are processed by hoosegow and the rest are passed through to the docker api.